app/src/androidTest/java/com/github/bgabriel998/softwaredevproject/CollectionActivityTest.java

```diff
@@ -58,12 +65,44 @@ public void TestToolbarBackButton(){
         assertSame(testRule.getScenario().getResult().getResultCode(), Activity.RESULT_CANCELED);
     }
 
-    /* Test that pressing the collected item the view changes to MountainActivity */
+    /* Test that all elements in list view are at correct place and contains correct data */
     @Test
-    public void TestCollectedItem(){
-        ViewInteraction button = Espresso.onView(withId(R.id.collected));
-        button.perform(ViewActions.click());
-        // Catch intent
-        intended(IntentMatchers.hasComponent(MountainActivity.class.getName()));
+    public void TestContentOfListView(){
```

Comment:
When we will display the actual data this test will have to be reworked, right? Maybe we should add a TODO comment? Also, what will we display when no internet connection is available? Maybe we can test this case as of now.